### PR TITLE
Add new filter mockup

### PR DESCRIPTION
**Pull request by maethu**
- [x] Provide [filter.html](https://github.com/OneGov/plonetheme.onegov/blob/mle-filter-mockup/plonetheme/onegov/resources/filter.html) mockup for two different filter implementations
- [x] Provide the according styles
- [x] Extend readme with sample image and link to zg.ch search for a example implementation.

### CODE REVIEW

**Comment by maethu** [comment]
@jone

**Comment by maethu** [comment]
@bierik  your feedback would be appreciated!

**Comment by jone** [comment]
:+1: 
@bierik ?

File: plonetheme/onegov/resources/filter.html [hunk 1]
```diff
@@ -0,0 +1,364 @@
...
+                    <span>Topic</span>
+                    <ul>
+                      <li><a>Option 1</a></li>
+                      <li><a>Option 2</a></li>
+                      <li><a>Option 3</a></li>
+                    </ul>
+                  </div>
+                  <div class="filter">
+                    <span>Last change</span>
+                    <ul style="display: block;">
```
Comment (bierik) [inline plonetheme/onegov/resources/filter.html]: Why inline styling?

Comment (maethu) [inline plonetheme/onegov/resources/filter.html]: For the mockup... if your opening the site you don't have to click around, instead it shows directly the dropdown (check image). display:block ist what the js part als does.

**Comment by maethu** [comment]
@bierik please take another look!

**Comment by bierik** [comment]
:+1: look good

**Pull request merged**